HTML/Answers/07.cookies-session-and-localstorage-storage-difference.md

```diff
@@ -0,0 +1,71 @@
+# Cookies
+
+- **Limited** storage space **4KB**(~4096B)
```

Comment:
I think the below representation would be nice to have

**Limited** storage space **4096 bytes / ~4 kb**